spec/tool-bar-spec.js

```diff
@@ -76,9 +115,8 @@ describe('Tool Bar package', () => {
         toolBar = workspaceElement.querySelector('.tool-bar');
       });
 
-      it('by third-party packages', () => {
-        expect(toolBarAPI).toBeDefined();
-        expect(toolBarAPI.group).toBe('specs-tool-bar');
+      afterEach(() => {
+        toolBarAPI.unregister();
```

Comment:
This might be redundant, as it appears the specs is deactivating the package after each run.